src/classes/Srt.js

```diff
@@ -0,0 +1,35 @@
+import fs from 'fs'
+import {
+	isEmptyFile,
+} from '../utils/files'
+
+export class Srt {
+	constructor({
+		basePath = './',
+		fileName,
+	} = {}) {
+		this.basePath = basePath
+		this.fileName = fileName
+		this.writeStream = fs.createWriteStream(
```

Comment:
This actually does not instantly create the file -- it starts that process.

This is an issue because:

1. we may invoke appendPayload on the write stream before waiting for the `ready` event
2. we may invoke delete (or fstat) on the write stream before the file is actually created

Since the write stream is not actually exposed anywhere we may want to replace this with something like `openSync`.  We don't *need* it to be a writeStream.